common/djangoapps/student/management/commands/export_staff_users.py

```diff
@@ -0,0 +1,72 @@
+from __future__ import absolute_import, print_function
+
+import logging
+from datetime import datetime, timedelta
+from django.core.management.base import BaseCommand
+from django.conf import settings
+from django.core.mail.message import EmailMultiAlternatives
+from django.template.loader import get_template
+from os import remove
+
+from djqscsv import write_csv
+from openedx.core.djangoapps.content.course_overviews.models import CourseOverview
+from student.models import CourseAccessRole
+
+logger = logging.getLogger(__name__)
+
+
+class Command(BaseCommand):
+
+    help = """
+    This command will export a csv of all users who have logged in within the last seven days and
+    have staff access role in active courses (Courses with end date in the future).
+    """
+
+    subject = 'Staff users CSV'
+    to_addresses = ['aazam@edx.org']
+    from_address = settings.DEFAULT_FROM_EMAIL
+    txt_template_path = 'email/export_staff_users.txt'
+    html_template_path = 'email/export_staff_users.html'
+    interval = '7 days'
```

Comment:
`interval` Should be calculated from the command line arg.

Reply:
@awaisdar001 I am using both dates for logging that is why I have not moved it to a new method